_inc/client/components/jetpack-notices/index.jsx

```diff
@@ -154,7 +154,7 @@ export const DevModeNotice = React.createClass( {
 				devModeType += __('your site URL lacking a dot (e.g. http://localhost).');
 			}
 
-			const text = __('Currently in {{a}}Development Mode{{/a}} VIA ' + devModeType + '{{br/}}Some features are disabled.',
+			const text = __('Currently in {{a}}Development Mode{{/a}} Via ' + devModeType + '{{br/}}Some features are disabled.',
```

Comment:
I'm not entirely sure what you had in mind. Would this work for you? 859b70987a01ad65e4ab3fb93827182d3c3c5d07

That's a lot of repetition though.